### DISCUSSION TITLE
Cannot verify message

### DISCUSSION THREAD

**Discussion by AlexanderEpolite** [openpgpjs/openpgpjs#1615] [Q&A]
Hello, I have the following:
```js
const a = await verify({
    format: "utf8",
    message: await readCleartextMessage({
        cleartextMessage: answer
    }),
    verificationKeys: await readKey({
        armoredKey: pubkey,
    })
});
```

Where `answer` and `pubkey` are strings.

I get this error before compiling when hovering over `message`:
![image](https://user-images.githubusercontent.com/88310235/226206211-67d053af-fa19-49c4-92d1-1408582cda8e.png)

and if I force it to run, I get this:
```
Error: No cleartext signed message.
    at exports.readCleartextMessage (/home/alex/gpgtestthing/node_modules/openpgp/dist/node/openpgp.min.js:16:139026)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async WebSocket.<anonymous> (/home/alex/gpgtestthing/dist/index.js:70:34)
```

I'm guessing there's something small I'm missing, but I can't find out what it is.  Thank you in advance!

**Comment by larabr** [comment]
Are you sure `answer` is a cleartext message and not a standard message? What's the armored header, `BEGIN PGP SIGNED MESSAGE`?
